packages/dds/tree/src/test/tablePerformanceTestUtilities.ts

```diff
@@ -0,0 +1,190 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
```

Comment:
For reviewers: This file can be skip 'cause this change will go away after https://github.com/microsoft/FluidFramework/pull/24881 being merged.